homeassistant/components/overkiz/services.yaml

```diff
@@ -0,0 +1,26 @@
+set_cover_position_and_tilt:
+  target:
+    entity:
+      integration: overkiz
+      domain: cover
```

Comment:
set_cover_position_and_tilt is only supported by vertical cover.  
You should add a supported_features here with a value of set_cover_position_and_tilt and add this to the list of supported_features on vertical_cover